### DISCUSSION TITLE
Upgrading Theia Version.

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#9893] [Q&A] [answered]
Hi All,
I'm planing to upgrade theia version to v1.16.0. Is replacing all the dependencies that involve @theia to v1.16.0 enough ? I'm new to this. 

Thanks in Advance

**Comment by vince-fugnitto** [comment, marked as answer]
@maheshrajrp yes, updating the version of the `@theia` dependencies and rebuilding will be enough (it will update the lockfile if you have one present). Please take note of any possible breaking changes in the [changelog](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md) if you use custom extensions, and any possible [migration steps](https://github.com/eclipse-theia/theia/blob/master/doc/Migration.md) as well.

**Reply by maheshrajrp** [reply]
Thanks, @vince-fugnitto .
